src/main/resources/resource/WebGui/app/service/views/WebXrGui copy.html

```diff
@@ -0,0 +1,193 @@
+<!doctype html>
```

Comment:
this file probably shouldn't be checked in...

Reply:
yeah .. last couple of commits i'll try to strip out the vertx & webxr stuff